.github/workflows/build_test.yml

```diff
@@ -0,0 +1,57 @@
+name: simple-kbs tests
+
+on: 
+  push:
+    branches: [ main, staging ]
+  pull_request:
+    branches: [ main, staging ]
```

Comment:
In attestation-agent we have:

```
on: [push, pull_request, create]
```

why change it here?

Specifically for pull requests we want to test branches that are not the main branches (to verify that the PR code is passing).

Reply:
It looks like the AA CI job is set to only run on pull requests, though. Running stuff whenever we create a branch might be a bit annoying for people who fork the repo and create feature branches. It seems like the important thing is to make sure we test the PR when it is submitted, which I think this will do.

Reply:
👍